lighthouse-core/audits/audit.js

```diff
@@ -33,6 +35,27 @@ class Audit {
   }
 
   /**
+   * Computes and clamps the score between 0 and 100 based on the measured value, and the two control
+   * points on the log-normal distribution (point of diminishing returns and the median value).
+   *
+   * @param {number} measuredValue
+   * @param {number} diminishingReturnsValue
+   * @param {number} medianValue
+   * @return {number}
+   */
+  static computeScore(measuredValue, diminishingReturnsValue, medianValue) {
```

Comment:
since not all audits compute their score this way, would it make more sense for this to live in `statistics.js` still? and/or a more specific method name